interface-spec.html

```diff
@@ -146,12 +146,24 @@ <h3><dfn>Term</dfn> interface</h3>
       <dfn>value</dfn> is refined by each interface which extends Term.
     </p>
     <p>
-      <dfn>equals()</dfn> returns <code>true</code> if and only if <code>other</code> has the same
-      <code>termType</code> and the same contents (as defined by concrete subclasses).
+      <dfn id="term-equals">equals()</dfn> returns <code>true</code>
+      when called with parameter <code>other</code>
+      on an object <code>term</code> if and only if
+      all of the conditions below hold:
+    </p>
+    <ul>
+      <li><code>other</code> is <em>neither</em> <code>null</code> nor <code>undefined</code>;</li>
+      <li><code>term.termType</code> is the same string as <code>other.termType</code>;</li>
+      <li><code>other</code> follows the additional constraints of the specific <code>Term</code> interface implemented by <code>term</code>
+        (e.g., <a href="#namednode-interface"><code>NamedNode</code></a>,
```

Comment:
Just `<a>NamedNode</a>` is enough to link internal. Respec detects that it's an interface and will format it like `code`.